apis/palette_api/generate_identifiers.py

```diff
@@ -0,0 +1,129 @@
+import json
```

Comment:
Maybe worth putting this file in a `/scripts` or something dir? not sure

Reply:
not intending to leave this here long term - i think it's worth puzzling out the problem more thoroughly in `data-pipedream` discussions